src/osx/carbon/font.cpp

```diff
@@ -1035,21 +1235,174 @@ void wxNativeFontInfo::SetStrikethrough(bool strikethrough)
 
 void wxNativeFontInfo::UpdateNamesMap(const wxString& familyName, CTFontDescriptorRef descr)
 {
-    if ( gs_FontFamilyToPSName.find(familyName) == gs_FontFamilyToPSName.end() )
+    if (gs_FontFamilyToPSName.find(familyName) == gs_FontFamilyToPSName.end())
     {
-        wxCFStringRef psName( (CFStringRef) CTFontDescriptorCopyAttribute(descr, kCTFontNameAttribute));
+        wxCFStringRef psName((CFStringRef)CTFontDescriptorCopyAttribute(descr, kCTFontNameAttribute));
         gs_FontFamilyToPSName[familyName] = psName.AsString();
     }
 }
 
 void wxNativeFontInfo::UpdateNamesMap(const wxString& familyName, CTFontRef font)
 {
-    if ( gs_FontFamilyToPSName.find(familyName) == gs_FontFamilyToPSName.end() )
+    if (gs_FontFamilyToPSName.find(familyName) == gs_FontFamilyToPSName.end())
     {
-        wxCFRef<CTFontDescriptorRef> descr(CTFontCopyFontDescriptor( font ));
+        wxCFRef<CTFontDescriptorRef> descr(CTFontCopyFontDescriptor(font));
         UpdateNamesMap(familyName, descr);
     }
 }
 
+// code that still may be useful
```

Comment:
I'd really prefer to keep it in a side branch instead of here, it's confusing to see commented out code like this and it appears in search results etc.

Reply:
cleanup committed